README.md

```diff
@@ -91,11 +91,11 @@ If using PostgreSQL, to administer the database manually (not essential, but use
 We provide project files to help you build and browse code using IntelliJ IDEA.
 If you wish to take advantage of this convenience, you should also install IntelliJ.
 
-
 Then:
 * Pull the repository
-* Set variables. Duplicate `/resources/variables-template.sh`, and overwrite with values suitable for your environment. Copy it to a convenient location and set an environmental variable VARIABLES_FILE to point to it using the full path. We recommend you set that up in .bashrc so that it is always definted correctly.
-* The favoured (and our current) approach is to use 'in memory' persistence, specified in `node.properties` in the working folder. However, postgres is an option. If using postgres, setup and run the db. Run `/bin/db/setup.sh`
+* Build the project using `mvn package -f code/core/pom.xml`
```

Comment:
The project is built with maven, whether you do that on command line or through the IDE.

When you do it on the command line, then you use the command `mvn package` from the project folder.

Not sure what that line is with the -f, i guess you can use that command if you're not IN the project folder already.